File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/VehicleProfileUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests;

import lombok.Data;

@Data
public class VehicleProfileUpdateRequest {
    private final String updatedVehicleBrand;
    private final String updatedVehicleVIN;
    private final String updatedVehicleRegistrationSign;
    private final String updatedVehicleRegistrationCertificate;
    private final String updatedVehicleOwnerFullName;
    private final String updatedVehicleOwnerResidentialAddress;
    private final String updatedVehicleInsurancePolicyNumber;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/VehicleProfileUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

data class VehicleProfileUpdateRequest(
    val updatedVehicleBrand: String,
    val updatedVehicleVIN: String,
    val updatedVehicleRegistrationSign: String,
    val updatedVehicleRegistrationCertificate: String,
    val updatedVehicleOwnerFullName: String,
    val updatedVehicleOwnerResidentialAddress: String,
    val updatedVehicleInsurancePolicyNumber: String
)
